scripts/vendor-boringssl.sh

```diff
@@ -145,6 +145,9 @@ if ! hash ${sed} 2>/dev/null; then
     exit 43
 fi
 
+echo "SAVING modulemap"
+cp "$DSTROOT/include/module.modulemap" "$TMPDIR"
```

Comment:
Rather than copy this back and forth, it would be better for us to just create it wholesale every time. This is what we do with the umbrella header, and it allows this script to avoid relying on any code being present in the repository at all.

Reply:
amended https://github.com/apple/swift-crypto/pull/69/commits/06bd913bdc8577e524d71b188889eb182a917586